lib/galaxy/jobs/runners/cli.py

```diff
@@ -104,7 +104,8 @@ def queue_job(self, job_wrapper):
             return
         # Some job runners return something like 'Submitted batch job XXXX'
         # Strip and split to get job ID.
-        external_job_id = stdout.strip().split()[-1]
+        submit_stdout_pieces = stdout.strip().split()
+        external_job_id = submit_stdout_pieces and submit_stdout_pieces[-1]
```

Comment:
Nitpicking, but I'd prefer if `external_job_id` is an empty string and not an empty list in this case:

```suggestion
        submit_stdout = stdout.strip()
        external_job_id = submit_stdout and submit_stdout.split()[-1]
```